documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -1,33 +1,32 @@
 [[raspi-config]]
 == `raspi-config`
 
-TIP: Raspberry Pi Desktop users can access a graphical version of this application at **Preferences** > **Raspberry Pi Configuration**. However, some advanced configuration is only available in `raspi-config`.
+`raspi-config` helps you configure your Raspberry Pi. Changes to `raspi-config` will affect xref:config_txt.adoc#what-is-config-txt[`/boot/firmware/config.txt`] and various Linux configuration files.
 
-`raspi-config` helps you configure your Raspberry Pi.  Available options may differ between Raspberry Pi models. To open the configuration tool, run the following command:
+=== Getting started
+
+To open the configuration tool from the desktop GUI, go to **Preferences** > **Raspberry Pi Configuration**.
+
+Alternatively, to access the configuration tool via the terminal:
```

Comment:
```suggestion
Alternatively, run the following command to access the configuration tool via the terminal:
```
Suggestion: Introducing command line snippets. This also helps contrast example output, bash that users should write into a file, and interactive snippets.